### DISCUSSION TITLE
How to update log level at runtime without restarting the docker container

### DISCUSSION THREAD

**Discussion by psathyavathy** [keycloak/keycloak#27693] [Archive]
How can we modify the log level at runtime in Quarkus distribution environment. #

docker compose file configured as below for  default logging,
      - KC_LOG=file,console
      - KC_LOG_LEVEL=INFO

I would to like to change the log level without stop/restart my docker container, is there any way?
Please provide your thoughts.
Reference: https://github.com/keycloak/keycloak/discussions/10414

**Comment by psathyavathy** [comment]
keycloak version used is 23.0.4

**Reply by shawkins** [reply]
@psathyavathy I don't know of a way to do that through keycloak. https://stackoverflow.com/questions/63250947/how-to-change-the-log-level-during-runtime-in-quarkus mentions exposing jax-rs endpoints for this - you could extrapolate other solutions from there. In particular you have to consider any runtime only changes will be undone on a container restart. They would need to be persistent and picked up again on restart.
